infra/README.md

```diff
@@ -0,0 +1,61 @@
+# Deploying Fair Data infrastructure with Terraform
+
+This guide outlines the steps to deploy the infrastructure components for the Fair Data finder. The following components will be deployed:
+
+- Front-end and back-end containers are deployed in an ECS (Elastic Container Service) service.
+- Back-end connects to a Postgres database in RDS (Relational Database Service).
+- Both services connect to an Opensearch database.
```

Comment:
Only the backend communicates with Postgres and Opensearch. Frontend only communicates with backend.